crates/avian3d/examples/debugdump_3d.rs

```diff
@@ -0,0 +1,17 @@
+//! Run with:
+//! `cargo run --example debugdump_3d > dump.dot  && dot -Tsvg dump.dot > dump.svg`
+
+use avian3d::debug_render::PhysicsDebugPlugin;
+use avian3d::prelude::*;
+use bevy::prelude::*;
+
+fn main() {
+    let mut app = App::new();
+
+    app.add_plugins((PhysicsPlugins::default(), PhysicsDebugPlugin::default()));
+    // Schedules of interest:
+    // - PhysicsSchedule
+    // - SubstepSchedule
```

Comment:
Attempting to make a graph for this schedule results in an error, I'll open an upstream issue.